shell/packages/sandstorm-ui-autocomplete-input/autocomplete-client.js

```diff
@@ -0,0 +1,210 @@
+window.UserContacts = new Mongo.Collection("userContacts");
```

Comment:
Why is this being added as a field of `window`? If you expect to need it outside of this package, it would be better to expose it via `api.export()`. But it looks like you aren't actually using it outside this package, so maybe it can just be `var UserContacts = ...`?